components/FlexibleForms/RepeaterGroupField.tsx

```diff
@@ -35,6 +38,8 @@ const RepeaterGroupField = ({
     touched: FormikTouched<FormikValues>;
   } = useFormikContext();
 
+  const [hidden, setHidden] = useState(hiddenOnStart ?? false);
```

Comment:
this seems to do the trick! though i wonder if we could do it in a stateless way by modifying line 82 on the `lib/utils/generateInitialValues.ts` file - the `getInitialValues` function. we could do something like:

```
if(field.initiallyBlank){
initialValues[field.id] = []
} else {
initialValues[field.id] = generateInitialValues(field.subfields || [], person),
}